iOS/ExposureNotification/source/ApiDefinition.cs

```diff
@@ -59,7 +59,7 @@ public enum ENStatus : long {
 		Unauthorized = 6,
 	}
 
-	[Introduced (PlatformName.iOS, 13, 7)]
+	[Introduced (PlatformName.iOS, 12, 5)]
```

Comment:
that might be even mode confusing in this case, as 13.5/6 won't have it either

Reply:
Good point. Let me revert these.